README.md

```diff
@@ -35,9 +35,20 @@
 
 ## Releasing OpenSearch
 
+### About OpenSearch releases and versions
+
+The OpenSearch Project releases as versioned bundles of OpenSearch, OpenSearch Dashboards, and the OpenSearch plugins. The OpenSearch Project [follows semantic versioning](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/). Software, such as Data Prepper, clients, and the Logstash output plugin, are versioned independently of the OpenSearch Project. They also may have independent releases from the main project distributions. In addition to the semantic versioning, the OpenSearch Project may release software under alpha, beta, release candidate, and generally available labels. The definition of when to use these labels is derived from [the Wikipedia page on Software release lifecycle](https://en.wikipedia.org/wiki/Software_release_life_cycle). Below is the definition of when to use each label.
```

Comment:
lowercase project

bundles -> distributions

It follows semantic versioning ...